### ISSUE TITLE
New Technique: radioactive tracer administration?

### ISSUE THREAD

**Issue by szafarnia** [HumanBrainProject/openMINDS_controlledTerms#423]
Is "radioactive tracer administration" the same technique as "contrast agent administration"? Or they should be registered separately? 

I searched a little bit and found following information:
While both techniques involve the administration of substances into the body, they serve different purposes and are used in different imaging modalities. Radioactive tracer administration focuses on studying physiological processes and metabolism, whereas contrast agent administration primarily enhances the visibility of structures or tissues in imaging studies. Therefore, I wonder if the should be registered separately?!

**Comment by tgbugs** [comment]
Can we have a bit more context? Also what is the tracer itself? Is it itself radioactive?

**Comment by apdavison** [comment]
One difference is that contrast agents increase an already existing contrast, whereas radioactive tracers _produce_ the signal (which you could perhaps think of as increasing the contrast from zero to a non-zero value)

**Comment by tgbugs** [comment]
I agree, but I assume there are cases where they localize based on existing differences in e.g. tissue properties.
